x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/styles.tsx

```diff
@@ -24,7 +24,7 @@ export const LABEL_PADDING_X = 15;
 export const LABEL_BORDER_WIDTH = 1;
 export const NODE_WIDTH = 90;
 export const NODE_HEIGHT = 90;
-const NODE_LABEL_WIDTH = 120;
+export const NODE_LABEL_WIDTH = 160;
```

Comment:
: for-later-consideration

I would like to see if we can use screen breakpoints from EUI to adjust the graph sizes.  Not right now, but this would make the graphs respond to screen size changes.

Reply:
in addition to the ability to zoom in and out we also adjust the size of the graph preview when flyout size changes using `ResizeObserver` (https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph/graph.tsx#L142-L149)

Then, these sizes are relative and not fixated

is that what you mean?

Reply:
Yep, that works also.  Disregard the comment.